internal/common/scale.go

```diff
@@ -33,6 +34,8 @@ func runScale(autoScalingSupported bool) {
 				Skip("autoscaling is not supported")
 			}
 
+			helper.SetResponsibleTeam(helper.TeamPhoenix)
```

Comment:
I believe this should be tenet, [because CAS is owned by tenet](https://github.com/giantswarm/cluster-autoscaler-app/blob/main/CODEOWNERS#L2).

Reply:
Yeah I wasn't sure about this one as auto-scaling is only a CAPA feature. I'm happy to move it to our team but I'm not sure if it would actually be changes we make that might cause this to fail. What do you think?

Reply:
I'm going to switch it to Tenet for now just to unblock this PR but please could you discuss it in Phoenix to see if y'all think it makes sense or not? :)